ql/src/test/queries/clientpositive/materialized_view_partitioned.q

```diff
@@ -60,3 +60,7 @@ ALTER MATERIALIZED VIEW partition_mv_3 REBUILD;
 ALTER MATERIALIZED VIEW partition_mv_3 REBUILD;
 
 SELECT * FROM partition_mv_3 where key = 238;
+
+DROP MATERIALIZED VIEW partition_mv_1;
+DROP MATERIALIZED VIEW partition_mv_2;
+DROP MATERIALIZED VIEW partition_mv_3;
```

Comment:
at some point you could probably add some cleanup to qtestutil for these around:

https://github.com/apache/hive/blob/69aec11e711293eaf5eb8a3fbb44905bc11841c9/itests/util/src/main/java/org/apache/hadoop/hive/ql/QTestUtil.java#L365

Reply:
Created a follow-up patch: https://issues.apache.org/jira/browse/HIVE-25906